introduction.md

```diff
@@ -2,3 +2,8 @@ I am using github for the first time
 code school videos are great 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?